common/lib/xmodule/xmodule/js/src/problem/edit.coffee

```diff
@@ -200,7 +200,8 @@ class @MarkdownEditingDescriptor extends XModule.Descriptor
 #}
 #
   @markdownToXml: (markdown)->
-    toXml = `function (markdown) {
+    explanation_title = gettext "Explanation"
```

Comment:
I'm fairly certain this needs to be a function call (`gettext("Explanation")`)

It would also be good to add a translator's comment, to explain the context in which this is used. Please see http://edx.readthedocs.org/projects/edx-developer-guide/en/latest/internationalization/i18n.html#coffeescript-files for more detail.